python/hail/linalg/matrix.py

```diff
@@ -20,6 +23,42 @@ def read(cls, path):
         return cls(Env.hail().distributedmatrix.BlockMatrix.read(
             hc._jhc, path))
 
+    @staticmethod
+    @handle_py4j
+    def _from_numpy_matrix(numpy_matrix, block_size):
+        """Create a BlockMatrix from the given numpy matrix.
```

Comment:
"Create a block matrix from a NumPy matrix."

I prefer official name when referring to software/language (e.g. NumPy, Java, Scala, Python).
And we use table, matrix table, block matrix, etc when referring to objects of these types in docs.